pkg/kubernetes/common.go

```diff
@@ -62,3 +70,14 @@ func GetDaprVersion(status []StatusOutput) string {
 	}
 	return daprVersion
 }
+
+func GetImageRegistry() (string, error) {
+	defaultImageRegistry, err := utils.GetDefaultRegistry(githubContainerRegistryName, dockerContainerRegistryName)
+	if err != nil {
+		return "", err
+	}
+	if defaultImageRegistry == githubContainerRegistryName {
```

Comment:
@pravinpushkar Shouldn't the GetDefaultRegistry method already return the right registry URI? Why are we again checking if it is GHCR and modifying it here?

Reply:
@mukundansundar `GetDefaultRegistry ` returns only the registry name as in GHCR or dockerhub. Since in standalone mode there are different images like dapr/dapr, dapr/3rdparty/redis etc the actual URI creation happens in later in standalone.go only.

Doc issue is there, will be creating the doc PR soon.